src/main/java/frc/robot/commands/HatchCommands/SendFlipperToLim.java

```diff
@@ -0,0 +1,29 @@
+package frc.robot.commands.HatchCommands;
+
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj.command.Command;
+import frc.robot.Robot;
+import frc.robot.RobotMap;
+
+public class SendFlipperToLim extends Command {
```

Comment:
Sigh.  I so wish we had a working limit switch for this.

Reply:
Right, this would be very nice to have. it should hopefully be fixed in the next week or so